File: 1083.py
# Introductory Problems : Missing Number
#  https://cses.fi/problemset/task/1083/

# We can obtain missing numbers by finding sum from 1->n and then 
# subtracting the sum of the numbers we have in the list. 

n = int(input())
a = list(map(int, input().split()))
n2 = sum(a)
n1 = (n*(n+1))//2
print(n1-n2)

File: 1083.cpp
// Introductory Problems
// https://cses.fi/problemset/task/1083/

#include <iostream>
using namespace std;
typedef long long lolo;
int main(){
    int n, a;
    cin >> n;
    lolo sum = 0;
    for (int i = 0; i < n-1; i++) {
        cin >> a;
        sum += a;
    }
    cout << lolo(n) * (n+1)/2 - sum << endl;
}